container-chains/templates/frontier/runtime/src/xcm_config.rs

```diff
@@ -288,7 +289,34 @@ impl pallet_asset_rate::AssetKindFactory<AssetId> for ForeignAssetBenchmarkHelpe
     }
 }
 
-pub type AssetId = u16;
```

Comment:
This should not be removed

Reply:
Yes this should be set to `u16`. I changed it for testing purposes to `u128` and I forgot to put it back again to its original value.